src/Microsoft.AspNet.Mvc.Xml/Formatters/FormattingUtilities.cs

```diff
@@ -1,6 +1,9 @@
 // Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
+using System.Collections.Generic;
+using System.Linq;
 #if ASPNET50
```

Comment:
This condition is present as the type `XsdDataContractExporter` is only present in desktop CLR. Does that answer your question or am i missing something?